malaria24/ona/models.py

```diff
@@ -757,6 +757,17 @@ def __unicode__(self):
         return u'%s (%s)' % (self.name, self.role)
 
 
+class InboundSMS(models.Model):
+    """
+    An SMS sent to the system.
+    """
+    sender = models.CharField(max_length=255)
+    content = models.CharField(max_length=255)
+    message_id = models.CharField(max_length=255)
```

Comment:
I wonder if we should make this the ID of the model? To avoid having an integer and a UUID as identifiers? Either way, this should be a unique field, we can't have two messages with the same ID.